requirements.txt

```diff
@@ -61,7 +62,8 @@ queuelib==1.4.2
 requests==2.13.0
 scikit-learn==0.18.1
 scipy==0.19.0
-Scrapy==1.3.3
+scons==2.3.4
+Scrapy==1.1.1
```

Comment:
Can you check your version? It seems weird to revert to an older version of `Scrapy`.